lib/archethic_web/graphql_schema/resolver.ex

```diff
@@ -41,15 +41,40 @@ defmodule ArchethicWeb.GraphQLSchema.Resolver do
     t2 = Task.async(fn -> get_transaction_content(address) end)
 
     with {:ok, {:ok, genesis_address}} <- Task.yield(t1),
-         {:ok, {:ok, transaction_content}} <- Task.yield(t2) do
+         {:ok,
+          {:ok,
+           %{
+             "name" => name,
+             "supply" => supply,
+             "symbol" => symbol,
+             "type" => type,
+             "properties" => properties
```

Comment:
Here properties may not be present and if we create a token without properties it raise a no match error.
I suggest to use something like `Map.get(map, "properties", []) to return an empty list by default

Reply:
Ok